vita3k/gui/src/user_management.cpp

```diff
@@ -686,8 +686,9 @@ void draw_user_management(GuiState &gui, EmuEnvState &emuenv) {
         ImGui::TextColored(GUI_COLOR_TEXT, "%s", lang["name"].c_str());
         ImGui::SetCursorPos(INPUT_NAME_POS);
         ImGui::PushItemWidth(INPUT_NAME_SIZE);
-        if (ImGui::InputText("##user_name", temp.name.data(), SCE_NP_ONLINEID_MAX_LENGTH))
-            temp.name = temp.name.data(); // Hack
+        char user_name_buf[SCE_NP_ONLINEID_MAX_LENGTH + 1]{};
```

Comment:
Should not works att all here...
This code request global value, look more code
If char is set from nothing any time nothing works, my code works fine

Reply:
I have tested, and like exepted, no works att all 😕
Just create char like that is wrong
Temp.name is already temp value and is set in one function, for set default name, or current in edit
Like i said, is more’complicated you thinks, is better noctouch it, code perfectly works here